src/daemon/remote.proto

```diff
@@ -7,6 +7,7 @@ package dist_clang.daemon.proto;
 message Remote {
   optional base.proto.Flags flags = 1;
   optional string source          = 2;
+  optional uint32 emitter_version = 3 [ default = 100 ];
```

Comment:
Let it be simply "version" - and the default value should be real, i.e. the current version (790.0). Also the real version is complex and consists of two numbers: better to reflect it as a separate message type, like:

    message Version {
      optional uint32 major = 1 [ default = 790 ];
      optional uint32 minor = 2 [ default = 0];
    }

This message type should be shared with a its counterpart in Absorber message.